.github/workflows/cron_update_modules/script.php

```diff
@@ -126,4 +126,4 @@ function fetchVersionComposerLock(): array {
     . '| How to test?      | N/A' . PHP_EOL
     . '| Possible impacts? | N/A' . PHP_EOL
 ;
-echo $pullRequestBody;
+putenv("SCRIPT_PULL_REQUEST_BODY=$pullRequestBody");
```

Comment:
You will have the same problem as previous :thinking: The text is not encapsulate.